generators/app/templates/src/redux/Auth/actions.js

```diff
@@ -1,60 +0,0 @@
-import { push } from 'connected-react-router';
```

Comment:
all this behaviour was lost right? It would be nice to add cards to recreate all of this in the corresponding components.

Reply:
Let's do it when we implement the Login component